Tests/FluentAssertions.Specs/Types/TypeSelectorSpecs.cs

```diff
@@ -498,6 +499,38 @@ public void When_selecting_types_that_are_not_classes_it_should_return_the_corre
                 .And.Contain(typeof(NotOnlyClassesEnumeration));
         }
 
+        [Fact]
+        public void When_selecting_types_that_are_value_types_it_should_return_the_correct_types()
+        {
+            // Arrange
+            Assembly assembly = typeof(InternalEnumValueType).GetTypeInfo().Assembly;
```

Comment:
`GetTypeInfo()` should not be necessary anymore since we droppped some old frameworks in FA 6.0.0.
I see that we use this elsewhere in this test, but feel free to remove them all.